File: android/app/src/main/java/net/ipv64/kivop/dtos/PosterServiceDTOs/CreatePosterPositionDTO.kt
package net.ipv64.kivop.dtos.PosterServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class CreatePosterPositionDTO (
    var posterId : UUID?,
    var latitude : Double,
    var longitude : Double,
    var responsibleUsers : List<UUID>,
    var expiresAt : LocalDateTime,
)

File: DTOs/Sources/PosterServiceDTOs/CreatePosterPositionDTO.swift
import Foundation

public struct CreatePosterPositionDTO: Codable {
    public var latitude: Double
    public var longitude: Double
    public var responsibleUsers: [UUID]
    public var expiresAt: Date
    
    public init( latitude: Double, longitude: Double, responsibleUsers: [UUID], expiresAt: Date) {
        self.latitude = latitude 
        self.longitude = longitude 
        self.responsibleUsers = responsibleUsers
        self.expiresAt = expiresAt
    }
}


